naucse/utils/models.py

```diff
@@ -1,8 +1,36 @@
 from collections import OrderedDict
 from pathlib import Path
-import sys
 
 import yaml
+from arca import Task, Arca
+from werkzeug.local import LocalProxy
+
+import naucse.utils.routes
+
+_arca = None
+
+
+@LocalProxy
+def arca():
+    global _arca
+    if _arca is not None:
+        return _arca
+
+    _arca = Arca(settings={"ARCA_BACKEND": "arca.backend.CurrentEnvironmentBackend",
+                           "ARCA_BACKEND_CURRENT_ENVIRONMENT_REQUIREMENTS": "requirements.txt",
+                           "ARCA_BACKEND_VERBOSITY": 2,
+                           "ARCA_BACKEND_APK_DEPENDENCIES": ["libffi-dev"],
+                           "ARCA_BACKEND_KEEP_CONTAINER_RUNNING": True,
+                           # https://docs.travis-ci.com/user/pull-requests/#Pull-Requests-and-Security-Restrictions
+                           # "ARCA_BACKEND_PUSH_TO_REGISTRY_NAME": "docker.io/mikicz/naucse",
```

Comment:
Once we enable this, what happens with pull-requests? Will it fail Travuis build?

Don't forget to update this to the naucse user.